### ISSUE TITLE
Rayray55

### ISSUE THREAD

**Issue by Rayray3581** [klesun/deep-assoc-completion#213]
What does this mean can you spell it out for me in layman's terms please had somebody hack my phone his name is DeShawn Williams and I'm trying to press charges can you guys please help me out they took it all my money they took it on my pictures I need help please God bless

**Comment by klesun** [comment]
Hi. What does your issue have to do with this plugin? It's a plugin for IDE called Phpstorm, it has nothing to do with mobile phones.
